# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../seamonkey/glibc-2.12.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- comm-1.9.1/mozilla/toolkit/crashreporter/google-breakpad/src/common/linux/file_id.cc.vanilla	2011-05-15 16:29:03.000000000 +0200
+++ comm-1.9.1/mozilla/toolkit/crashreporter/google-breakpad/src/common/linux/file_id.cc	2011-05-15 16:29:54.000000000 +0200
@@ -40,6 +40,9 @@
 #include <sys/mman.h>
 #include <string.h>
 #include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
 
 #include "common/linux/file_id.h"
 #include "common/md5.h"
